### DISCUSSION TITLE
Random Survival Forest in lifelines

### DISCUSSION THREAD

**Discussion by phaniparsa** [CamDavidsonPilon/lifelines#1283] [General]
Hello All,

I've gone through lifelines documentation and don't see Random Survival Forest(RSF) implementation.

Was it ever under consideration to add RSF to lifelines?

Thanks!

**Comment by CamDavidsonPilon** [comment]
Hi @phaniparsa, 

No, I never considered it. Lifelines is designed for linear survival models. You can try scikit-survival, I believe they have RSF.

**Reply by phaniparsa** [reply]
cool, thank you for the response :)
